### PR TITLE
[PL-1104] - Prospective storyscripter gets feeling of automation in the logs

### PR DESCRIPTION
**Pull request by Arinono**
As a prospective storyscripter,
When I click deploy,
I see the deployment logs incrementally appear,
So I get the impression of the platform doing something cool for me

## Acceptance
**Given** I open an example
**Then** I don’t see the logs automatically appear
**And** When I click the Deploy button
**Then** I see the logs appear incrementally

## Blocker
~Need to merge this one first~
~https://github.com/storyscript/playground/pull/2~ Done ✅ 

## Notes

https://storyscript.atlassian.net/browse/PL-1104

playground-w2

### CODE REVIEW

**Comment by Arinono** [comment]
> However, the
> 
> ```
>   - http
>   - redis
> ```
> 
> appears incorrectly under `Configured 1 story`.

Addressed here 5470219

**Comment by Arinono** [comment]
> Additionally, the header doesn't seem to quite match up with the design:
> 
> **Design**
> 
> image
> 
> **Real**
> 
> image
> 
> It looks like there is a difference in size of the logo, and the spacing between the logo and deploy button.

Reduced the spacing between the logo and the button to be closer the the design:
Design spacing is `90px`, I previously rounded to `96px`. It's now rounded to `88px`.

The logo size seems smaller indeed, but matches the design:
- 82x16px for the text
- 18x18px for the logo

**Comment by williammartin** [comment]
> The logo size seems smaller indeed, but matches the design

Let's move forward with this then and if @aydaoz wants to address it later she can.

**Comment by aydaoz** [comment]
will reach to Jean and Aurelien on slack about it👌🏼

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -15,22 +15,69 @@
         color="text-indigo-logo"
         height="14"
         width="82"
-        class="ml-2 mt-1"
+        class="ml-1 mt-1"
```
Comment (JeanBarriere) [inline src/components/Navbar.vue]:
```suggestion
        class="ml-2 mt-1"
```

File: tailwind.config.js [hunk 2]
```diff
@@ -74,7 +74,9 @@ module.exports = {
       },
       spacing: {
         initial: 'initial',
-        'fit-content': 'fit-content'
+        'fit-content': 'fit-content',
+        '14': '3.5rem', // 56px
```
Comment (JeanBarriere) [inline tailwind.config.js]:
```suggestion
```

File: tests/unit/store/modules/Logs.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  describe('getters', () => {
+    describe('releasesCount', () => {
+      expect.assertions(1)
+      it('should return the value', () => {
+        const value = store.getters.getReleasesCount
+        expect(value).toEqual(0)
+      })
+    })
+  })
```
Comment (JeanBarriere) [inline tests/unit/store/modules/Logs.spec.ts]: missing mutation test

File: tailwind.config.js (hunk 2 above)
Comment (Arinono) [reply tailwind.config.js]: We are using it fir the navbar

File: src/components/Navbar.vue (hunk 1 above)
Comment (Arinono) [reply src/components/Navbar.vue]: Done

**Pull request merged**